app/component/map/popups/ParkAndRideHubPopup.js

```diff
@@ -0,0 +1,22 @@
+import Relay from 'react-relay';
+import ParkAndRidePopup from './ParkAndRidePopup';
+import withProps from 'recompose/withProps';
+
+export default Relay.createContainer(withProps(props => ({
+  realtime:
+    props.facilities.reduce((prev, current) => prev && (!current || current.realtime), true),
```

Comment:
Maybe these could be written without reduce using some more specific functions.
I.e. filtering nulls and then summing.